release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,22 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
+* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
+* Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.
+
+[id="console-supports-installing-specific-operator-versions"]
+===== Support for installing specific operator versions in the web console
+
+With this update, the console operator builds a set of custer `nodeOperatingSystems` types so only operators that can be installed render. You can then choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the operator will immediately update back to the latest version on the channel.
```

Comment:
So the first sentence is not related to the `Support for installing specific operator versions in the web console` feature. Its rather a separate functionality, where the console-operator is gathering the OS types of each cluster nodes and based on that creates a list of supported OS types on the cluster. That list is then passed to the console, so it will only render operators, in the operator hub, which are supporting that OS type.

Reply:
Thanks I wasn't sure on that! I have added a separate entry.